packages/lit-dev-content/.eleventy.js

```diff
@@ -143,22 +143,34 @@ ${content}
     return url.substring(0, url.length - extension.length);
   });
 
+  const sortDocs = (a, b) => {
+    if (a.fileSlug == 'docs') {
+      return -1;
+    }
+    if (a.fileSlug < b.fileSlug) {
+      return -1;
+    }
+    if (b.fileSlug < a.fileSlug) {
+      return 1;
+    }
+    return 0;
+  };
+
   const docsByUrl = new Map();
-  eleventyConfig.addCollection('docs', function (collection) {
+
+  eleventyConfig.addCollection('docs-v1', function (collection) {
+    const docs = collection.getFilteredByGlob('site/docs/v1/**').sort(sortDocs);
+    for (const page of docs) {
+      docsByUrl.set(page.url, page);
+    }
+    return docs;
+  });
+
+  eleventyConfig.addCollection('docs-v2', function (collection) {
     const docs = collection
       .getFilteredByGlob('site/docs/**')
-      .sort(function (a, b) {
-        if (a.fileSlug == 'docs') {
-          return -1;
-        }
-        if (a.fileSlug < b.fileSlug) {
-          return -1;
-        }
-        if (b.fileSlug < a.fileSlug) {
-          return 1;
-        }
-        return 0;
-      });
+      .filter((page) => !page.url.includes('v1'))
```

Comment:
Does that glob not support negation, like `.getFilteredByGlob('site/docs/!(v1)/**')`?

Reply:
Done (though had to do `.getFilteredByGlob(['site/docs/*', 'site/docs/!(v1)/**'])`)